GOVERNANCE.md

```diff
@@ -1,131 +1,188 @@
-* Help the Project build connections and partnerships;
-* Can be part of the Admin team for the project on GitHub
-
-#### Conflict of Interest
-In the event of any conflict of interest (a Committee Member, their family member, or someone with whom the Committee Member has a close academic or employment relationship is involved in a decision), the Committee Member must immediately notify other Committee Members. The Committee Members will be asked to recuse themselves from ongoing conversations and decision processes regarding the Transaction.  
-
-#### Current list of Advisory Committee members
-* Jan Ainali
-* Julie Faure-Lacroix
-* Esther Jackson
+**Current list of Advisory Committee members**
+- Jan Ainali [@Ainali](https://github.com/Ainali) and on the [forum](https://forum.openrefine.org/u/ainali/summary)
+- Julie Faure-Lacroix [@jfaurelacroix](https://github.com/jfaurelacroix) and on the [forum](https://forum.openrefine.org/u/jfaurel/summary)
+- Esther Jackson [@ej2432](https://github.com/ej2432) and on the [forum](https://forum.openrefine.org/u/ej2432/summary)
 
 ### Project Manager
-OpenRefine's Project Manager (paid position) works closely with OpenRefine’s advisory and steering committees and with its communities of users and contributors.
+OpenRefine's Project Manager (paid position) works closely with OpenRefine's advisory committee, Core Dev Groups and OpenRefine users and contributors.
 
 The Project Manager:
-* Helps to improve OpenRefine's governance and community diversity
-* Helps build community-driven structures to formalize OpenRefine’s roadmap and keep it up to date
-* Assists in finding and secure new sources of funding
-* Supports the Advisory Committee in OpenRefine’s day-to-day operations.
+- Helps to improve OpenRefine's governance and community diversity
+- Helps build community-driven structures to formalize OpenRefine's roadmap and keep it up-to-date
+- Assists in finding and securing new sources of funding
+- Supports the Advisory Committee in OpenRefine's day-to-day operations; this may include supervising paid project staff.
 
-The current Project Manager is [Martin Magdinier](https://forum.openrefine.org/u/Martin/summary)
-## Fiscal sponsorship: Code For Science and Society
+The current Project Manager is Martin Magdinier [@magdmartin](https://github.com/magdmartin) and on the [forum](https://forum.openrefine.org/u/Martin/summary).
+
+# OpenRefine Policies
+Introduction to Governance Policies
+
+The **Code of Conduct Committee, Advisory Committee, and Core Developer Group** hold greater authority within the OpenRefine project, with the ability to represent and make commitments on its behalf. To maintain the community’s trust, these groups are held to a higher standard of transparency and accountability in their decision-making processes.
+This includes clear and open practices for:
+- Member Election 
+- Requesting and managing funds
+- Managing conflicts of interest
+
+## Member Election
```

Comment:
This **Member Election** concerns itself with Committers, Core Developer Group, and Code of Conduct Committee members which is part of OpenRefine Governance (not Polices), and henceforth should be moved to after the **Release Manager** section to detail how the Governance groups are elected.

Advisory Committee elections and policies can remain in the **OpenRefine Policies** section.
Heck, I'd even go so far as to NOT call the section **OpenRefine Policies** but instead **Advisory Committee Policies** please.

Reply:
We agree here. I think the conversation is mostly about how we structure the document. I am ok with either:
1) Having the voting part in the core governance document (and removing mention of committers and Core Dev Group from the policy section)
2) Keeping things as is, such as mentioning the Committers group on line 103. 

I'll be interested in having other opinions on how to structure this best; @Ainali, maybe?

Reply:
I am keeping things as is to avoid more edits in the document. Again, I am happy to revisit my position.

Reply:
It could stay here, but since the elections touch upon the groups we mention higher up, it makes sense to me to move this section up there.

Reply:
This was discussed during t[the April 10 AC](https://forum.openrefine.org/t/april-10-2024-advisory-committee/2230), and the consensus was to move it to the governance section, which is done with my latest commit.